src/teuthology_api/services/helpers.py

```diff
@@ -96,3 +99,19 @@ def get_token(request: Request):
         detail="You need to be logged in",
         headers={"WWW-Authenticate": "Bearer"},
     )
+
+
+async def isAdmin(username, token):
+    TEAM_MEMBER_URL = (
+        f"https://api.github.com/orgs/ceph/teams/{ADMIN_TEAM}/memberships/{username}"
+    )
+    async with httpx.AsyncClient() as client:
+        headers = {
+            "Authorization": "token " + token,
+            "Accept": "application/json",
+        }
+        response_org = await client.get(url=TEAM_MEMBER_URL, headers=headers)
+        response_org_dic = dict(response_org.json())
+        if response_org_dic.get("state") == "active":
```

Comment:
if "state" doesn't exists then is there a chance response_org_dic = [ ] ? and will somehow stop the program?

Reply:
I did a quick check and doesn't look like it would throw an error:
```
>>> dict([])
{}
```
But I can add more error handling, thanks!